docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md

```diff
@@ -13,4 +13,5 @@ To develop IoT applications with SmartThings&trade;:
 
 - [Create a SmartThings&trade; project](rt-create-smartthings.md).
 - [Build the SmartThings&trade; project](rt-build-smartthings.md).
+- [Manage SmartThings™ certificates](rt-certificate-smartthings.md).
```

Comment:
Please change to SmartThings&trade;.

Reply:
Sure!

Reply:
I'm sorry for letting you be confused. GitHub displays \&trade; to &trade;.
I mean you change from SmartThings™ to SmartThings\&trade;, so I left a comment only here.